renku_notebooks/api/schemas/config_server_options.py

```diff
@@ -36,16 +36,10 @@ class Meta:
     lfs_auto_fetch = fields.Bool(required=True)
 
 
-class CloudStorageServerOption(Schema):
-    """Used to indicate in the server_options endpoint which types of cloud storage is enabled."""
+class ServerOptionsEndpointResponse(ServerOptionsChoices):
+    """Used to serialize the server options sent out through the server_options endpoint."""
 
-    s3 = fields.Nested(
+    cloudstorage = fields.Nested(
```

Comment:
This requires a helm chart change. Just a reminder.

Reply:
there is a corresponding pr in the main repo